src/components/text-indent/_macro.njk

```diff
@@ -0,0 +1,5 @@
+{% macro onsTextIndent(params) %}
+    <div class="text-indent">
+        {{ params.text | safe }}
```

Comment:
Could be good to set this up so it can be utilised using [call](https://mozilla.github.io/nunjucks/templating.html#call)

```suggestion
        {{ caller() if caller else params.text | safe }} 
```